openwisp_network_topology/urls.py

```diff
@@ -5,6 +5,6 @@
 
 urlpatterns = [
     url(r'^accounts/', include('openwisp_users.accounts.urls')),
-    url(r'^api/', include(api)),
+    url(r'^api/v1/', include(api)),
```

Comment:
this is a backward incompatible change, can you add a note in CHANGES.rst please?